miden/tests/integration/operations/io_ops/constant_ops.rs

```diff
@@ -46,7 +46,7 @@ fn push_many() {
     test.expect_stack(&expected);
 
     // --- push hexadecimal values with period separators between values ----------------------
-    let asm_op = format!("{}.0xA.0x64.0x3E8.0x2710.0x186A0", base_op);
+    let asm_op = format!("{}.0x0A.0x64.0x03E8.0x2710.0x0186A0", base_op);
```

Comment:
nit: shouldn't `0xA` be allowed? this looks like a valid hex descriptor that should be parsed to decimal `10`. Its of course a design decision to now allow possibly ambiguous descriptions, but most of languages accepts `0xA`

Reply:
I was asking Bobbin about it: https://github.com/maticnetwork/miden/pull/562#discussion_r1032785548 and we decided that it was better to make all the strings of even length for the sake of portability

Reply:
I remember that while work in JS (or maybe node.js), parsing hex strings required to be aligned on full bytes. Not sure if that's still the case, but I think it is fine to be conservative here. If a hex value is provided, it is likely to be for strings representing multiples of bytes (rather than half-bytes).